controllers/agentconfig_controller.go

```diff
@@ -191,8 +191,12 @@ func (r *AgentConfigReconciler) createEmptyPluginVolume(ctx context.Context, log
 	if err != nil {
 		return nil, false, err
 	}
-
-	if pluginPVC != nil || tempPVC != nil {
+	// If there is an exising claim then return it. This can be either the pluginPVC or a tempPVC depending on where in the process
```

Comment:
Will we ever hit this step with a PVC that is stuck in setup/failed setup?

Reply:
We shouldn't hit this because the creation is handled separately and this is just checking for if a PVC exists. We're not actually managing the contents of the PV. If it failed in creation then it wouldn't be able to get here.